install-server.sh

```diff
@@ -192,9 +192,12 @@ install_helm() {
 }
 
 update_helm_dependencies() {
-    echo "Updating Helm dependencies..."
-    helm repo add bitnami https://charts.bitnami.com/bitnami >/dev/null
-    helm repo update >/dev/null
+    local download_chart=${DOWNLOAD_CHART:-true}
+    if [[ "$download_chart" == true ]]; then
+        echo "Updating Helm dependencies..."
+        helm repo add bitnami https://charts.bitnami.com/bitnami >/dev/null
+        helm repo update >/dev/null
+    fi
```

Comment:
nitpick: consider reversing the if condition and returning early to keep the happy path outside the if block.